.travis.yml

```diff
@@ -18,6 +18,7 @@ before_install:
 install:
   - npm --production=false install
 script:
+  - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then audit-ci --moderate; fi
```

Comment:
why not equal to true? 🤔

Reply:
I think it's because `TRAVIS_PULL_REQUEST` is either a number or a string "false". I can see we are using this pattern everywhere in our code and it's also suggested here https://www.npmjs.com/package/audit-ci